pytest_fxa/plugin.py

```diff
@@ -48,12 +48,25 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize('fxa_urls', envs, indirect=True)
 
 
+def fxa_cleanup(account, fxa_client, fxa_account):
+    logger = logging.getLogger()
+    try:
+        account.clear()
+        fxa_client.destroy_account(fxa_account.email, fxa_account.password)
+        logger.info('Removed: {}'.format(fxa_account))
+    except ClientError as e:
+        # 'Unknown Account' error is ok - account already deleted
+        # https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-format
+        if e.errno not in [102]:
+            raise
+
+
 @pytest.fixture
 def fxa_client(fxa_urls):
     return Client(fxa_urls['authentication'])
 
 
-@pytest.fixture
+@pytest.fixture(scope='session')
```

Comment:
Is there a reason for this change? It looks like it isn't causing any test failures, so it might be okay, but I just wanted to check to see if this is intentional.

Reply:
I think I ```ctrl z```'d it back.